### PR TITLE
docs: update README.md and SECURITY.md

### PR DESCRIPTION
**Pull request by KaoImin**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Which docs this PR relation**:

Ref #

**Which toolchain this PR adaption**:

No Breaking Change

**Special notes for your reviewer**:

### CI Switch

ci-runs-only: [Code Format]

### CODE REVIEW

**Review by kelisi129 — DISMISSED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Security Policy
+
+This project is still under development, the primary goal at this stage is to implement features but we also take security very seriously. This document defines the policy on how to report vulnerabilities and receive updates when patches to those are released.
```
Comment (linnnsss) [inline SECURITY.md:3]: This is an ongoing project. While we mainly focus on implementing features, security is also a top priority. This document explains the policies on how to report vulnerabilities and receive updates when related patches are released.

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+[Join Nervos Security Team!](mailto:careers@nervos.org)
```
Comment (linnnsss) [inline SECURITY.md]: Join the Nervos Security Team!

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Reporting a vulnerability
+
+All security bugs should be reported by sending email to [Nervos Security Team \<security@nervos.org>](mailto:security@nervos.org). Please encrypt your mail using GPG with the following public key.
```
Comment (linnnsss) [inline SECURITY.md:9]: To report any security bugs, send an email to the Nervos Security Team. Please encrypt your email using GPG with the following public key:

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+* Unique ID: `Nervos Security Team <security@nervos.org>`
+* Fingerprint: C3D9 CF8A 1450 D04B 997E  4E31 6CBD D93A 0C9F 6BCD
+* Import from keys.openpgp.org: [0C9F6BCD](https://keys.openpgp.org/search?q=security@nervos.org)
+
+This will deliver a message to Nervos Security Team who handle security issues. Your report will be acknowledged within 24 hours, and you'll receive a more detailed response to your email within 72 hours indicating the next steps in handling your report.
```
Comment (linnnsss) [inline SECURITY.md:15]: Nervos Security Team will receive a message and handle the issue. Your report will be acknowledged within 24 hours, and you'll receive a more detailed response within 72 hours informing you of the next steps.

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+After the initial reply to your report the security team will endeavor to keep you informed of the progress being made towards a fix and full announcement.
```
Comment (linnnsss) [inline SECURITY.md:17]: The security team will keep you updated on the progress towards the fix and full announcement following the initial reply to your report.

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## Disclosure process
+
+1. Security report received and is assigned a primary handler. This person will coordinate the fix and release process. Problem is confirmed and all affected versions is determined. Code is audited to find any potential similar problems.
```
Comment (linnnsss) [inline SECURITY.md:21]: 1. Newly reported security issue will be assigned to a primary handler who coordinates the fix and release process. The issue and all the affected versions will be examined; the code will also be audited.

File: SECURITY.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+2. Fixes are prepared for all supported releases. These fixes are not committed to the public repository but rather held locally pending the announcement.
```
Comment (linnnsss) [inline SECURITY.md:22]: Fixes will be prepared for all supported releases. These fixes will not be committed to the public repository but rather held locally until the announcement is made.

File: SECURITY.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+3. A suggested embargo date for this vulnerability is chosen. This notification will include patches for all supported versions.
```
Comment (linnnsss) [inline SECURITY.md:23]: A suggested embargo date for this vulnerability will be given. This notification includes patches for all the supported versions.

File: SECURITY.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+4. On the embargo date, the [Nervos security mailing list](https://groups.google.com/u/0/a/nervos.org/g/security-mailing-list) is sent a copy of the announcement. The changes are pushed to the public repository. At least 6 hours after the mailing list is notified, a copy of the advisory will be published on Nervos community channels.
```
Comment (linnnsss) [inline SECURITY.md:24]: On the embargo date, the [Nervos security mailing list](https://groups.google.com/u/0/a/nervos.org/g/security-mailing-list) will send a copy of the announcement. The changes will be pushed to the public repository. At least 6 hours after the announcement is posted, a copy of the advisory will be published on the Nervos community channels.

File: SECURITY.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+This process can take some time, especially when coordination is required with maintainers of other projects. Every effort will be made to handle the bug in as timely a manner as possible, however it's important that we follow the release process above to ensure that the disclosure is handled in a consistent manner.
```
Comment (linnnsss) [inline SECURITY.md:26]: This process can take some time, especially when coordination with maintainers of other projects is required. The bugs will be addressed as quickly as possible, and we must follow the release process above to ensure that the disclosure is handled consistently.

File: SECURITY.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+## Receiving disclosures
+
+If you require prior notification of vulnerabilities please subscribe to the [Nervos Security mailing list](https://groups.google.com/u/0/a/nervos.org/g/security-mailing-list). The mailing list is very low traffic, and it receives the public notifications the moment the embargo is lifted.
```
Comment (linnnsss) [inline SECURITY.md:30]: You can subscribe to Nervos Security's mailing list if you would like to be notified about vulnerabilities in advance. The mailing list has low traffic, and it receives public notifications only when the embargo is lifted.

File: SECURITY.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+If you have any suggestions to improve this policy, please send an email to [Nervos Security Team](mailto:security@nervos.org).
```
Comment (linnnsss) [inline SECURITY.md:32]: If you have any suggestions to improve this policy, please send an email to Nervos Security Team.

File: README.md [hunk 13]
```diff
@@ -1,27 +1,57 @@
 # Axon
 
+[![version](https://img.shields.io/github/v/release/axonweb3/axon?sort=semver)](https://github.com/axonweb3/axon/releases) [![license](https://img.shields.io/github/license/axonweb3/axon)](https://github.com/axonweb3/axon/blob/main/LICENSE) ![status-check](https://github.com/axonweb3/axon/actions/workflows/web3_compatible.yml/badge.svg?branch=main)  ![contributors](https://img.shields.io/github/contributors/axonweb3/axon)
+
 > Axon is still in active development and a work in progress.
```
Comment (linnnsss) [inline README.md]: Axon is in active development/ is a work in progress. (保留一个即可)

File: README.md [hunk 14]
```diff
@@ -1,27 +1,57 @@
...
 
-## About Axon
+Axon is a high-performance layer 2 framework with native cross-chain function.  Built on top of the [Overlord](https://github.com/nervosnetwork/overlord) consensus protocol and the P2P network [Tentacle](https://github.com/nervosnetwork/tentacle), Axon supports hundreds of nodes and achieves thousands of TPS. Axon is also EVM-compatible with well-developed toolchains. It performs high interoperability, meaning deployed applications can achieve convenient cross-chain communication.
+
+
+## Highlights
+
+### About developer-friendly
```
Comment (linnnsss) [inline README.md]: ### DApps development made easy

File: README.md [hunk 15]
```diff
@@ -1,27 +1,57 @@
...
+
+Axon is compatible with [Ethereum](https://ethereum.org) so that all of the develop utilities can be used on it directly. And the [Axon CLI](https://github.com/axonweb3/axon-devops/axon-cli) provides an all-in-one client which includes initialization, DevOps, cross chain request and so on. If you consider seeing is believing, there is a [15 minutes tutorial]() that will lead you to build your own chain and deploy a tiny application.
```
Comment (linnnsss) [inline README.md]: Axon is 100% compatible with [Ethereum](https://www.notion.so/cryptape/Pure-Draft-11333a1cd81f40839496b4de80d317de), and the all-in-one [Axon CLI](https://github.com/axonweb3/axon-devops/axon-cli) implements all aspects of building, launching, and maintaining dApps, such as initialization, DevOps, and cross-chain requests.

You will be able to build your own chain and deploy your first dApp on Axon after watching this 15 minutes tutorial.

File: README.md [hunk 16]
```diff
@@ -1,27 +1,57 @@
...
+
+### Native cross-chain communication
+
+Openness and mobility are the foundation of social development, so is blockchain. Cross-chain derive the taking liquidity precisely. Axon satisfy native cross-chain ability without attach to a bridge. Each of the chain based Axon can cross to [CKB](https://www.nervos.org), any other Axon based chain, and [IBC](https://ibcprotocol.org) compatible chains. Axon will embed more cross-chain protocol in future.
+
+## Roadmap
+
+Most of the infrastructure has done and some substantial features to be developed are as below:
```
Comment (linnnsss) [inline README.md]: As most of the infrastructure has been done, our next major steps will be building the following substantial features:

File: README.md [hunk 17]
```diff
@@ -1,27 +1,57 @@
...
+
+## Highlights
+
+### About developer-friendly
+
+Axon is compatible with [Ethereum](https://ethereum.org) so that all of the develop utilities can be used on it directly. And the [Axon CLI](https://github.com/axonweb3/axon-devops/axon-cli) provides an all-in-one client which includes initialization, DevOps, cross chain request and so on. If you consider seeing is believing, there is a [15 minutes tutorial]() that will lead you to build your own chain and deploy a tiny application.
+
+### Native cross-chain communication
+
+Openness and mobility are the foundation of social development, so is blockchain. Cross-chain derive the taking liquidity precisely. Axon satisfy native cross-chain ability without attach to a bridge. Each of the chain based Axon can cross to [CKB](https://www.nervos.org), any other Axon based chain, and [IBC](https://ibcprotocol.org) compatible chains. Axon will embed more cross-chain protocol in future.
```
Comment (linnnsss) [inline README.md]: Axon encourages openness and mobility across blockchains. Without any bridge, Axon eases cross-chain communication and enhances liquidity. Each Axon-based chain can communicate with any other Axon-based chain, the layer 1 [CKB](https://www.nervos.org), and any other [IBC](https://ibcprotocol.org) compatible chains.

File: README.md [hunk 18]
```diff
@@ -1,27 +1,57 @@
...
+
+## Roadmap
+
+Most of the infrastructure has done and some substantial features to be developed are as below:
 
-Axon is a sidechain framework based on [Muta](https://github.com/nervosnetwork/muta) with UDT token staking on CKB.
-Dapp developers can use Axon toolkit to create and distribute chain-specific UDT on CKB. Chain token holders can stake UDT to become an Axon chain validator or just deposit UDT into Axon chain for use in Dapps.
-Axon will support [interoperability 2.0](https://medium.com/nervosnetwork/blockchain-abstraction-and-interoperability-2-0-eea98d81b7b6) and be 100% Ethereum compatible, like [Godwoken](https://github.com/nervosnetwork/godwoken).
+1. Be compatible with [IBC](https://github.com/cosmos/ibc) protocol.
+2. Implement the original homologous cross chain protocol.
```
Comment (linnnsss) [inline README.md]: Implement the original homologous cross-chain protocol. (我看大部分都用的 cross-chain)

File: README.md [hunk 19]
```diff
@@ -1,27 +1,57 @@
...
+3. Implement some other cross chain protocol.
```
Comment (linnnsss) [inline README.md]: Implement more cross-chain protocols.

File: README.md [hunk 20]
```diff
@@ -1,27 +1,57 @@
...
-$ cargo run --package axon --bin axon -- --config=./devtools/chain/config.toml --genesis=./devtools/chain/genesis_single_node.json
+```bash
+git clone https://github.com/axonweb3/axon.git
+cd axon && cargo build --release
 ```
-You can also use [axon-devops](https://github.com/axonweb3/axon-devops) to deploy a multi-node chain.
+
+## Toolchain
+
+Apart from the framework, axon has corresponding toolchains:
```
Comment (linnnsss) [inline README.md]: Apart from the framework, Axon also provides these toolchains:

File: README.md [hunk 21]
```diff
@@ -1,27 +1,57 @@
...
+
+- [Axon Faucet](https://github.com/axonweb3/axon-faucet): faucet for the chain based Axon.
+- [Axon Explorer](https://github.com/Magickbase/blockscan): a blockchain explorer for the chain based Axon.
+- [Axon DevOps](https://github.com/axonweb3/axon-devops): includes some utilities such as monitor, benchmark tool and so on.
+
+All the toolchains above can be dictated by **[Axon CLI](https://github.com/axonweb3/axon-cli) which is a exquisite and deft command line interface**. You can use Axon CLI to do anything related Axon.
```
Comment (linnnsss) [inline README.md]: All the toolchains above can be implemented by **[Axon CLI](https://github.com/axonweb3/axon-cli)**, an exquisite and deft command line interface that makes everything Axon-related easier.

File: README.md [hunk 22]
```diff
@@ -1,27 +1,57 @@
...
+git clone https://github.com/axonweb3/axon.git
+cd axon && cargo build --release
 ```
-You can also use [axon-devops](https://github.com/axonweb3/axon-devops) to deploy a multi-node chain.
+
+## Toolchain
+
+Apart from the framework, axon has corresponding toolchains:
+
+- [Axon Faucet](https://github.com/axonweb3/axon-faucet): faucet for the chain based Axon.
```
Comment (linnnsss) [inline README.md]: [Axon Faucet](https://github.com/axonweb3/axon-faucet): the faucet for the chain-based Axon.

File: README.md [hunk 23]
```diff
@@ -1,27 +1,57 @@
...
+- [Axon Explorer](https://github.com/Magickbase/blockscan): a blockchain explorer for the chain based Axon.
```
Comment (linnnsss) [inline README.md]: [Axon Explorer](https://github.com/Magickbase/blockscan): a blockchain explorer for the chain-based Axon.

File: README.md [hunk 24]
```diff
@@ -1,27 +1,57 @@
...
+- [Axon DevOps](https://github.com/axonweb3/axon-devops): includes some utilities such as monitor, benchmark tool and so on.
```
Comment (linnnsss) [inline README.md]: - [Axon DevOps](https://github.com/axonweb3/axon-devops): includes several utilities, such as monitor and benchmark tool.

File: README.md [hunk 25]
```diff
@@ -1,27 +1,57 @@
...
+Please read the [CONTRIBUTING.md](./CONTRIBUTING.md) for details on code of conduct, and the process for submitting pull requests to us. And the security policy is described in [SECURITY.md](./SECURITY.md).
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](./LICENSE) file for details.
 
 ## Acknowledgment
 
-Axon is based on [Muta](https://github.com/nervosnetwork/muta). Sincerely thanks to everyone who [contributed code to Muta](https://github.com/nervosnetwork/muta/graphs/contributors).
+Axon is based on [Muta](https://github.com/nervosnetwork/muta). Sincerely thanks to everyone who [contributed code to Muta](https://github.com/nervosnetwork/muta/graphs/contributors).
```
Comment (linnnsss) [inline README.md]: Axon is based on [Muta](https://github.com/nervosnetwork/muta). Sincere thanks go to everyone who has [contributed code to Muta](https://github.com/nervosnetwork/muta/graphs/contributors).

File: README.md [hunk 26]
```diff
@@ -1,27 +1,57 @@
...
+
+- [Axon Faucet](https://github.com/axonweb3/axon-faucet): faucet for the chain based Axon.
+- [Axon Explorer](https://github.com/Magickbase/blockscan): a blockchain explorer for the chain based Axon.
+- [Axon DevOps](https://github.com/axonweb3/axon-devops): includes some utilities such as monitor, benchmark tool and so on.
+
+All the toolchains above can be dictated by **[Axon CLI](https://github.com/axonweb3/axon-cli) which is a exquisite and deft command line interface**. You can use Axon CLI to do anything related Axon.
+
+## Contributing
+
+Please read the [CONTRIBUTING.md](./CONTRIBUTING.md) for details on code of conduct, and the process for submitting pull requests to us. And the security policy is described in [SECURITY.md](./SECURITY.md).
```
Comment (linnnsss) [inline README.md]: Please read the [CONTRIBUTING.md](./CONTRIBUTING.md) for details on the code of conduct and the process of submitting pull requests. The security policy is described in [SECURITY.md](./SECURITY.md).

File: README.md [hunk 27]
```diff
@@ -1,27 +1,57 @@
...
+2. Implement the original homologous cross chain protocol.
+3. Implement some other cross chain protocol.
 
-As a layer-2 solution on [Nervos](https://www.nervos.org/), Axon can achieve thousands of TPS with the help of [Overlord](https://github.com/nervosnetwork/overlord), a high-performance consensus.
-This offers Nervos DApp developers one more option. Godwoken is perfect for scenarios that involve low throughput and high value, such as DeFi, where security is usually more important than speed.
-When it comes to consumer-facing applications such as decentralized games or social networks, Axon offers the best performance.
+## Install
 
-## Quick Start
+The [release page](`https://github.com/axonweb3/axon/release`) provides the compiled binary, and if you want to build from source code, please make sure that [rust](https://www.rust-lang.org/), [clang](http://clang.org/), [openssl](https://www.openssl.org/), [m4](https://www.gnu.org/software/m4/) have already been installed and then execute the following command:
```
Comment (linnnsss) [inline README.md]: The [release page](`https://github.com/axonweb3/axon/release`) provides the compiled binary. If you want to build from source code, please make sure that you have already installed [rust](https://www.rust-lang.org/), [clang](http://clang.org/), [openssl](https://www.openssl.org/), [m4](https://www.gnu.org/software/m4/). Then execute the following command:

File: README.md [hunk 28]
```diff
@@ -1,27 +1,57 @@
 # Axon
 
+[![version](https://img.shields.io/github/v/release/axonweb3/axon?sort=semver)](https://github.com/axonweb3/axon/releases) [![license](https://img.shields.io/github/license/axonweb3/axon)](https://github.com/axonweb3/axon/blob/main/LICENSE) ![status-check](https://github.com/axonweb3/axon/actions/workflows/web3_compatible.yml/badge.svg?branch=main)  ![contributors](https://img.shields.io/github/contributors/axonweb3/axon)
+
 > Axon is still in active development and a work in progress.
 
-## About Axon
+Axon is a high-performance layer 2 framework with native cross-chain function.  Built on top of the [Overlord](https://github.com/nervosnetwork/overlord) consensus protocol and the P2P network [Tentacle](https://github.com/nervosnetwork/tentacle), Axon supports hundreds of nodes and achieves thousands of TPS. Axon is also EVM-compatible with well-developed toolchains. It performs high interoperability, meaning deployed applications can achieve convenient cross-chain communication.
```
Comment (linnnsss) [inline README.md]: Axon is a high-performance layer 2 framework with native cross-chain function.  Built on top of the [Overlord](https://github.com/nervosnetwork/overlord) consensus protocol and the P2P network [Tentacle](https://github.com/nervosnetwork/tentacle), Axon supports hundreds of nodes and achieves thousands of TPS. Axon is also EVM-compatible with well-developed toolchains. Its high interoperability facilitates cross-chain communication among dApps.

File: README.md (hunk 18 above)
Comment (kelisi129) [reply README.md]: cross-chain 是一个单词，之前 Notion 里的我直接改掉了

File: README.md (hunk 17 above)
Comment (kelisi129) [reply README.md]: 建议保留第一句”Openness and mobility are the foundation of social development, so is blockchain. “
Axon satisfies native cross-chain functionality without any bridge.... Axon will integrate with more cross-chain protocol in the future.

File: README.md [hunk 29]
```diff
@@ -1,48 +1,63 @@
...
+## What is Axon
 
-Axon is a high-performance layer 2 framework with native cross-chain function.  Built on top of the [Overlord](https://github.com/nervosnetwork/overlord) consensus protocol and the P2P network [Tentacle](https://github.com/nervosnetwork/tentacle), Axon supports hundreds of nodes and achieves thousands of TPS. Axon is also EVM-compatible with well-developed toolchains. It performs high interoperability, meaning deployed applications can achieve convenient cross-chain communication.
+Axon is a high-performance layer 2 framework with native cross-chain function.  Built on top of the [Overlord](https://github.com/nervosnetwork/overlord) consensus protocol and the P2P network [Tentacle](https://github.com/nervosnetwork/tentacle), Axon supports hundreds of nodes and achieves thousands of TPS. Axon is also EVM-compatible with well-developed toolchains. Its high interoperability facilitates cross-chain communication among dApps.
 
 
 ## Highlights
 
-### About developer-friendly
+### DApps development made easy
```
Comment (kelisi129) [inline README.md]: 建议还是用 “About developer-friendly”：“Developer Friendly” is a slam on tools and applications that should have been more user-friendly than they actually turned out. + 一般 protocols 用 developer-friendly 做形容词的比较多

**Pull request merged**